src/pages/public_registration/action_accordions/SupportAccordion.tsx

```diff
@@ -30,6 +31,8 @@ export const SupportAccordion = ({ registration, supportTicket, addMessage, refe
   const dispatch = useDispatch();
   const { t } = useTranslation();
   const user = useSelector((store: RootState) => store.user);
+  const userCristinIdentifier = user && getIdentifierFromId(user.nvaUsername);
+  const userIsTicketOwner = supportTicket?.owner === userCristinIdentifier;
```

Comment:
Den første der gir vel ikke cristinIdentifier, men hele username som det er, siden `getIdentifierFromId` splitter på `/` som ikke fins i username? `owner` er vel også username mener jeg, så tror heller ikke vi ønsker å ha cristinIdentifier her. Men om vi gjør det burde vi bruke `getIdentifierFromId(user.cristinId)`
```suggestion
  const userIsTicketOwner = supportTicket?.owner === user.nvaUsername;
```